IntroToCSharp/Pages/Activities/TheTypeSystem.razor

```diff
@@ -0,0 +1,36 @@
+@inherits CaptainCoder.Page
+@page "/activities/the-type-system"
+
+<MudText Typo="Typo.h4" Style="font-weight:bold">The Type System</MudText>
+
+<Section Title="Introduction">
+    <MudText>As we learned in the <MudLink Href="/activities/variables">Variables</MudLink> lesson, each variables must
+        have a <Keyword Word="type"></Keyword> that describes its <Keyword Word="data"></Keyword>. There are <Keyword
+            Word="14"></Keyword> different types of data that programmers can use. Defining data types for our variables
+        allows the computer to know how to store it while our program runs.
+    </MudText>
+</Section>
+<Section Title="Integers">
+    <MudText>
+        There are 8 different integer types that each use a certain amount of bytes. The amount of bytes in a type
```

Comment:
Perhaps list the different types so that they can see them? You don't need to explain what they all do but just a brief description could be good.